pom.xml

```diff
@@ -92,6 +110,68 @@
         <version>1.7.30</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>${junit.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.assertj</groupId>
+        <artifactId>assertj-core</artifactId>
+        <version>${assertj.core.version}</version>
```

Comment:
I don't think the Maven Property `assertj.core.version` is needed anymore: now that the version is only defined at `dependencyManagement` level, there is a single usage of the variable, so we can inline it. 

The same seems to apply to 
- commons.lang.version
- junit.version
- mockito.version
- stax2.api.version
- staxmate.version

Reply:
`junit.version` was needed but the other no. I've removed them